### PR TITLE
Improvements on external radios

### LINKED ISSUE #284: External radios issues
**Description:**
- After our test session, the following points must be addressed:

- [x] Prevent external user from open the radio GUI.

### PR DESCRIPTION
**Pull request by TheMagnetar**
**When merged this pull request will:**
- Disables the option of opening the radio by the external user.
- Closes #284.

### CODE REVIEW

File: addons/ace_interact/fnc_radioChildrenActions.sqf [hunk 1]
```diff
@@ -59,4 +59,9 @@ if ((_idx > -1) and (_idx < 3)) then {
 _action = ["acre_mptt_assign", _txt, "", {}, {true}, {_this call FUNC(radioPTTChildrenActions);}, _params] call ace_interact_menu_fnc_createAction;
 _actions pushBack [_action, [], _target];
 
-_actions
+if (GVAR(connectorsEnabled)) then {
+    _action = ["acre_connectors", "Connectors", "\idi\acre\addons\ace_interact\data\icons\connector4.paa", {}, {true}, {_this call FUNC(generateConnectors);}, _params] call ace_interact_menu_fnc_createAction;
```
Comment (jonpas) [inline addons/ace_interact/fnc_radioChildrenActions.sqf]: Stringtable?

Comment (TheMagnetar) [reply addons/ace_interact/fnc_radioChildrenActions.sqf]: oops, mixed  with racks...

File: addons/sys_prc117f/fnc_openGui.sqf [hunk 2]
```diff
@@ -17,9 +17,15 @@
 #include "script_component.hpp"
 
 TRACE_1("OPENING GUI", _this);
+params ["_radioId", "", "", "", ""];
+
+// Prevent radio from being opened if it is externally used.
```
Comment (jonpas) [inline addons/sys_prc117f/fnc_openGui.sqf]: No final dot. In other files too.

File: addons/sys_external/stringtable.xml [hunk 3]
```diff
@@ -45,5 +45,8 @@
             <English>Stop using %1 from %2</English>
             <Russian>Перестать использовать %1 из %2</Russian>
         </Key>
+        <Key ID="STR_ACRE_sys_external_noOpenGUI">
+            <English>Radio is used externally. Configuration must be made by the owner.</English>
```
Comment (jonpas) [inline addons/sys_external/stringtable.xml]: Does that even fit standard message box height?

**Review by jonpas — CHANGES_REQUESTED** [review]

File: addons/sys_external/stringtable.xml (hunk 3 above)
Comment (TheMagnetar) [reply addons/sys_external/stringtable.xml]: yes.

Comment (jonpas) [reply addons/sys_external/stringtable.xml]: What about `Radio used externally. Only owner may configure.`?

Comment (TheMagnetar) [reply addons/sys_external/stringtable.xml]: Yeah, much better!

**Review by jonpas — APPROVED** [review]

**Review by Soldia1138 — APPROVED** [review]
Looking good for me. Resolve that merge conflict and we can merge

**Pull request merged**